tests/Transports.Subscriptions.WebSockets.Tests/WebSocketWriterPipelineTests.cs

```diff
@@ -220,12 +220,12 @@ public async Task should_support_correct_case()
                 resultingJson);
         }
 
-        private WebSocketWriterPipeline CreateWebSocketWriterPipeline(IContractResolver contractResolver)
+        private WebSocketWriterPipeline CreateWebSocketWriterPipeline(NamingStrategy namingStrategy)
         {
             return new WebSocketWriterPipeline(_testWebSocket, new DocumentWriter(
                 new JsonSerializerSettings
                 {
-                    ContractResolver = contractResolver,
+                    ContractResolver = new ExecutionResultContractResolver() { NamingStrategy = namingStrategy },
                     NullValueHandling = NullValueHandling.Ignore,
                     Formatting = Formatting.None
                 }));
```

Comment:
Bit nasty that change demonstrated here. I guess for people that are just using defaults, they'll be all good, but otherwise it's a breaking one.

Perhaps we need to put something in the readme around configuration of the `IDocumentWriter`?

Reply:
It would be good.

Reply:
On the other hand, that's something that should belong in the core project, rather than here.